docs/source/index.rst

```diff
@@ -0,0 +1,131 @@
+NuCypher
+========
+*A proxy re-encryption network to empower privacy in decentralized systems*
+
+.. image:: .static/img/nucypher_logo.svg
+    :width: 60%
+----
+
+.. image:: https://circleci.com/gh/nucypher/nucypher/tree/master.svg?style=svg
+    :target: https://circleci.com/gh/nucypher/nucypher/tree/master
+
+.. image:: https://img.shields.io/pypi/wheel/nucypher.svg
+    :target: https://pypi.org/project/nucypher/
+
+.. image:: https://img.shields.io/pypi/pyversions/nucypher.svg
+    :target: https://pypi.org/project/nucypher/
+
+.. image:: https://coveralls.io/repos/github/nucypher/nucypher/badge.svg?branch=master
+    :target: https://coveralls.io/github/nucypher/nucypher?branch=master
+
+.. image:: https://img.shields.io/discord/411401661714792449.svg
+    :target: https://discord.gg/M5B7Nh
+
+.. image:: https://img.shields.io/pypi/l/nucypher.svg
+    :target: https://www.gnu.org/licenses/gpl-3.0.html
+
+
+- GitHub https://www.github.com/nucypher/nucypher
+- Documentation https://nucypher.readthedocs.io/en/latest/
+- Website https://www.nucypher.com/
+
+
+Whitepapers
+-----------
+
+**Network**
+
+    https://github.com/nucypher/whitepaper/blob/master/whitepaper.pdf
+
+    *"NuCypher - A proxy re-encryption network to empower privacy in decentralized systems"*
+    *by Michael Egorov, David Nuñez, and MacLane Wilkison - NuCypher*
+
+
+**Economics**
+
+    https://github.com/nucypher/mining-paper/blob/master/mining-paper.pdf
+
+    *"NuCypher - Mining & Staking Economics"*
+    *by Michael Egorov, MacLane Wilkison - NuCypher*
+
+
+**Cryptography**
+
+    https://github.com/nucypher/umbral-doc/blob/master/umbral-doc.pdf
+
+    *"Umbral A Threshold Proxy Re-Encryption Scheme"*
+    *by David Nuñez - NuCypher*
+
+
+.. warning::
+
+   NuCypher is currently in the *Alpha* development stage and is **not** intended for use in production.
+
+
+.. toctree::
+   :maxdepth: 2
+
+
+Architecture
+------------
+
+.. toctree::
+   :maxdepth: 1
+
+   architecture/contracts
+   architecture/upgradeable_proxy_contracts
+
+
+Guides
+------
+
+.. toctree::
+   :maxdepth: 2
+
```

Comment:
We should think about the ordering of the guides - currently it feels a little disjointed. Perhaps something like (?):

guides/installation_guide
guides/federated_testnet_guide
guides/ursula_configuration_guide
guides/quickstart (since this seems to summarize steps from above)
guides/contribution_guide

Also, should we have these guides all under a single section eg. `Setup`? If so, then perhaps contribution_guide should be moved out of this section.

Reply:
@derekpierre's ordering suggestion seems good - although should quickstart be first?

Reply:
👍 - I guess the goal for quickstart was the have readers not have to read the entire installation document.

Reply:
Just noticed that the guides are sectioned off in the homepage - but I was referring to the navigation bar in the left-hand side. Is that possible (?)